NOTICE

```diff
@@ -0,0 +1,15 @@
+eddymotion
+Copyright 2021-present The NiPreps Developers <nipreps@gmail.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may find a copy of the License in the LICENSE file of the
+project root or obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
```

Comment:
NOTICE is not a repetition of the license banner, but clarifications about the licensing of the code (e.g., if in the past had other licenses, or if it derives from pre-exiting code)

```suggestion
Copyright The NiPreps Developers.

This product includes software developed by
the NiPreps Community (https://nipreps.org/).

```